quinn-proto/src/lib.rs

```diff
@@ -149,13 +148,8 @@ pub mod fuzzing {
     }
 }
 
-/// The QUIC protocol version implemented
-const VERSION: std::ops::RangeInclusive<u32> = 0xff00_001d..=0xff00_0020;
-
-/// Whether a 4-byte slice represents a supported version number
-fn is_supported_version(x: &[u8]) -> bool {
-    VERSION.contains(&u32::from_be_bytes(x.try_into().unwrap()))
-}
+/// The QUIC protocol version implemented.
+pub const DEFAULT_SUPPORTED_VERSIONS: std::ops::RangeInclusive<u32> = 0xff00_001d..=0xff00_0020;
```

Comment:
Here too, I feel like we should just use a `Vec` rather than a contiguous `Range`.

Reply:
heap allocations are not allowed in constants, and making it static requires a clone, which I don't think adds much advantage.

Reply:
A constant slice wouldn't require a heap allocation.